manifests/official/ec2/ebs-jessie-amd64-hvm.yml

```diff
@@ -14,20 +14,40 @@ image:
 system:
   release: jessie
   architecture: amd64
-  bootloader: extlinux
+  bootloader: grub
   charmap: UTF-8
   locale: en_US
   timezone: UTC
 volume:
   backing: ebs
   partitions:
-    type: none
+    type: msdos
     root:
       filesystem: ext4
       size: 8GiB
 packages:
   mirror: http://cloudfront.debian.net/debian
+  components:
+  - main
+  - contrib
+  - non-free
+  install:
```

Comment:
CLI, boto are to make the instance usable without requiring every boto to have to load these on launch; if an instance starts within an IAM Role for API keys and a security group that is highly restrictive (or indeed, a subnet within a VPC with no Internet access) then bootstrapping these basic libraries and utils becomes difficult. Eg, imagine if an instance is nly permitted (by an policy) that it is only allowed to make outbound HTTPS connections; then apt-transport-https is a hard dependency). We've shipped this for some time to enable the base image be subjected to stricter run-time environments.